source/community/reactnative/src/components/actions/select-Action.js

```diff
@@ -51,7 +45,39 @@ export class SelectAction extends React.Component {
 				this.toggleVisibilityForElementWithID(this.props.selectActionData.targetElements);
 				break;
 			default:
+				this.onCustomActionCalled();
 				break;
 		}
 	}
+
+	/**
+	 * @description Invoked for the custom action type
+	 */
+	onCustomActionCalled() {
+		let actionObject = {};
+		for (let key in this.payload) {
+			if (this.payload.hasOwnProperty(key)) {
```

Comment:
Isn't this condition redundant? Key is from payload. So this.payload.hasOwnProperty(key) will be true always.

Reply:
Resolved. Removed the method.